modules/kubernetes/src/main/java/org/apache/ignite/kubernetes/configuration/KubernetesConnectionConfiguration.java

```diff
@@ -138,6 +141,24 @@ public boolean getIncludeNotReadyAddresses() {
         return includeNotReadyAddresses;
     }
 
+    /**
+     * Specifies the port which is returned to the caller to use for service discovery
+     * defaults to 0
+     * @param discoveryPort Port to use for Kubernetes IP Finder
+     * @return {@code this} for chaining.
+     */
+    public KubernetesConnectionConfiguration setDiscoveryPort(int discoveryPort) {
+        this.discoveryPort = discoveryPort;
+        return this;
```

Comment:
Hi! Could you please align formatting with other public methods:
1. You should put a point in end of lines in all comments.
2. Add additional new line before `@param` and `return`.

Reply:
Done, via Pavel's suggestions.